wagn/lib/wagn/all.rb

```diff
@@ -1,2 +1,2 @@
-# backwards compatibility
-require 'wagn/application'
\ No newline at end of file
+require 'decko/engine'
+require 'wagn/application'
```

Comment:
I thought we were getting rid of wagn/all in new code but just supporting it still to keep old wagns working.  So (1) why the change? and (2) it definitely has to require the application (when I added my first comment it didn't)

I'm not worried about namespaces, I'm worried about dependencies. For the sake of argument, let's just use "decko" here and talk about dependencies as they will be in the future. 

1. decko/application will depend on decko/engine.  So there's no need to require both like this unless there's a load order issue.  There shouldn't be.
2. In general, I would expect that most high-level things in the decko gem (specs, scripts, etc) would require the application and most things in decko-rails would require just the engine.   

I know that, the application is now an extremely thin wrapper (no controllers, no routes, etc), so we might be able to get away with requiring the engine instead, but that would run the risk of ignoring application-level configuration options in decks.  So we may want to set up some situations like that for testing purposes.